### DISCUSSION TITLE
API Documentation

### DISCUSSION THREAD

**Discussion by ThePaulMcBride** [plausible/analytics#637] [Anything Else]
Hey folks. Loving Plausible so far.

Are there any docs for the api the dashboard uses? I run a job board and I'd like to be able to pull the stat for a specific page out from my server and share them with my customers without having to give them full access to all of my analytics.

Is something like that possible? I could work it out pretty easily by using the network inspector but I'd prefer if the API was well documented and public.

**Comment by metmarkosaric** [comment]
thanks @ThePaulMcBride, glad to hear that! we don't have a public api at this moment. we started working on it last week and it's the number 1 priority on our roadmap. when we release the first beta version, we'll let you know and we'll have the documentation ready ([the main thread is here](https://github.com/plausible/analytics/discussions/95)). what you see being shared on twitter etc, are people hacking and playing around. it's nothing officially supported or documented yet until we have the first beta version.

**Reply by ThePaulMcBride** [reply]
Thanks Marko!
